### DISCUSSION TITLE
Docs do not have a sidebar for simple navigation

### DISCUSSION THREAD

**Discussion by thecodeinfluencer** [resend/react-email#1364] [Ideas]
The docs pages do not have a sidebar. This makes it difficult to navigate to a specific page. 

You never really know what the sections are until you navigate through all of them. Also makes it difficult to capture newly published sections in the docs. 

The sidebar should be a basic feature for a documentation website. 

Please consider adding a sidebar or migrating the docs to [Docusaurus](https://docusaurus.io/)

**Comment by gabrielmfern** [comment]
We do have a sidebar. It's just not designed the same as Docusaurus.

> You never really know what the sections are until you navigate through all of them.

I'm not sure if I follow here, how would having a differently designed sidebar help with this?
I think you mean that the sections cant be navigated through on the left sidebar, if that's the case there is the table oif contents on the right for all pages.

**Reply by thecodeinfluencer** [reply]
I was not able to see the sidebar initially. I opened the docs on a private window and when I saw the sidebar, I actually recalled the docs having a sidebar. Why isn't the sidebar visible on one window and visible on the other (private).

![Screenshot 2024-03-14 at 19 58 47](https://github.com/resend/react-email/assets/38157066/976b15c6-db0e-4c57-a2c7-e520458765fc)

**Comment by thecodeinfluencer** [comment]
Found the issue!
Apparently this was being caused by an extension. [Dark Mode - Dark Reader for Сhrome](https://chromewebstore.google.com/detail/pjbgfifennfhnbkhoidkdchbflppjncb) in my case. Turned that off and everything is back to normal.
